docs/introduction.mdx

```diff
@@ -9,56 +9,120 @@ import SolanaIcon from '@site/src/assets/SolanaIcon'
+ 
+<Grid> 
+ 
+<GridLink 
+  to="https://docs.terra.lido.fi" 
+  title="Terra" 
+  subtitle="bLuna and stLuna" 
+  Icon={<TerraIcon />} 
+/> 
+ 
+<GridLink 
+  to="https://docs.solana.lido.fi" 
+  title="Solana" 
+  subtitle="stSOL" 
+  Icon={<SolanaIcon />} 
+/> 
+ 
+<GridLink 
+  to="https://docs.polygon.lido.fi" 
+  title="Polygon" 
+  subtitle="stMATIC" 
+  Icon={<PolygonIcon />} 
+/> 
+ 
+<GridLink 
+  to="https://docs.cosmos.lido.fi" 
+  title="Cosmos" 
+  subtitle="stAtom" 
+  Icon={<CosmosIcon />} 
+/> 
+ 
+<GridLink 
+  to="https://docs.kusama.lido.fi" 
+  title="Kusama" 
+  subtitle="stKSM" 
+  Icon={<KusamaIcon />} 
+/> 
+ 
+<GridLink 
+  to="https://docs.polkadot.lido.fi" 
+  title="Polkadot" 
+  subtitle="stDOT" 
+  Icon={<PolkadotIcon />} 
+/> 
+ 
+</Grid> 
+ 
 
-<GridLink
-  to="https://docs.polygon.lido.fi"
-  title="Polygon"
-  subtitle="stMATIC"
-  Icon={<PolygonIcon />}
-/>
+## Liquid staking 
 
-<GridLink
-  to="https://docs.cosmos.lido.fi"
-  title="Cosmos"
-  subtitle="stAtom"
-  Icon={<CosmosIcon />}
-/>
+### Problem statement
 
-<GridLink
-  to="https://docs.kusama.lido.fi"
-  title="Kusama"
-  subtitle="stKSM"
-  Icon={<KusamaIcon />}
-/>
+Traditionally, staking in PoS protocol based projects has been about locking one’s assets in one project for a long time and expecting a fixed, predetermined staking reward in return. While it guarantees the return on staked assets much like a bond, it also limits the opportunities of generating higher returns on those assets from the DeFi ecosystem. If you’ve staked all of your crypto holdings, you can’t invest or trade-in more profitable crypto pairs on exchanges. 
 
-</Grid>
+### Solution 
+ 
+Liquid staking allows using the staked crypto assets in other trading or investing opportunities to let the user get the best of both worlds - a reward on your staked assets, as well as the returns from new trading/investing opportunities that you spot. Liquid staking introduce various fundamental benefits by: 
+- Allowing users to earn staking rewards without fully locking their `st[token]`;  
+- Making it possible to earn rewards on as small a deposit as users want (i.e, Ethereum requires minimum 32 `ETH` staked);  
+- Providing the `st[token]` a building block for other applications and protocols (e.g., as collateral in lending or other trading DeFi solutions);  
+- Providing an alternative to exchange staking, self-staking, and other semi-custodial and decentralised protocols. 
+ 
+### Advantages over self staking and exchange staking 
+ 
+**Self-staking** is great, but it comes with some disadvantages. Setting up a validator node requires a pristine technical understanding; brings with it a minimum deposit of 32 ETH; slashing and offline penalties can get very severe if the staking is managed improperly; the staked amount is locked up for a significant period. Eventually, it leads to a small number of stakers, which is bad for any PoS ecosystem. 
```

Comment:
> Maybe 'solo staking' is a more widespread term to use, wdyt?

Definitely a more proper definition, will makes right away

> See: https://ethereum.org/en/staking/#how-to-stake-your-eth

A great comparison table is being attached there. Probably, it would make sense to do something similar in the future digging deeper in the features of liquid staking 🤔